### ISSUE TITLE
Can't seem to run the basic build - getting a java.io.FileNotFoundException 

### ISSUE THREAD

**Issue by sagistrauss** [DigitalPebble/storm-crawler#324]
Hey im trying to get a simple example running but when i build from the archetype and run 
`mvn clean compile exec:java -Dexec.mainClass=CrawlTopology -Dexec.args="-conf crawler-conf.yaml -local"`
i get an error, it complains that it is missing a file in `flux-core.jar!/resources`
now iv'e went into the jar and the folder is there. 
how can i go about troubleshooting this issue, please help.

**Comment by jnioche** [comment]
Thanks, I can reproduce the problem and will look into it. In the meantime, using the storm command should work as a workaround

**Comment by jnioche** [comment]
Here is the full stack trace

```
11334 [Thread-8] INFO  o.a.s.d.supervisor - Copying resources at jar:file:/home/julien/.m2/repository/org/apache/storm/flux-core/1.0.1/flux-core-1.0.1.jar!/resources to /tmp/2f218499-138a-4770-afe8-26ff1d92a125/supervisor/stormdist/crawl-1-1472032813/resources
11335 [Thread-8] ERROR o.a.s.event - Error when processing event
java.io.FileNotFoundException: Source 'file:/home/julien/.m2/repository/org/apache/storm/flux-core/1.0.1/flux-core-1.0.1.jar!/resources' does not exist
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1368) ~[storm-core-1.0.1.jar:1.0.1]
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1261) ~[storm-core-1.0.1.jar:1.0.1]
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1230) ~[storm-core-1.0.1.jar:1.0.1]
	at org.apache.storm.daemon.supervisor$fn__9359.invoke(supervisor.clj:1194) ~[storm-core-1.0.1.jar:1.0.1]
	at clojure.lang.MultiFn.invoke(MultiFn.java:243) ~[clojure-1.7.0.jar:?]
	at org.apache.storm.daemon.supervisor$mk_synchronize_supervisor$this__9078$fn__9096.invoke(supervisor.clj:582) ~[storm-core-1.0.1.jar:1.0.1]
	at org.apache.storm.daemon.supervisor$mk_synchronize_supervisor$this__9078.invoke(supervisor.clj:581) ~[storm-core-1.0.1.jar:1.0.1]
	at org.apache.storm.event$event_manager$fn__8630.invoke(event.clj:40) [storm-core-1.0.1.jar:1.0.1]
	at clojure.lang.AFn.run(AFn.java:22) [clojure-1.7.0.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
```

the `resources` directory does exist in the jar though.

**Comment by jnioche** [comment]
The same issue happens when calling the Flux topology from Maven

`mvn clean compile exec:java -Dexec.mainClass=org.apache.storm.flux.Flux -Dexec.args="--local crawler.flux"`

**Comment by jnioche** [comment]
interestingly the same issue happens when removing Flux from the dependencies but adding storm-crawler-tika

```
java.io.FileNotFoundException: Source 'file:/home/pebble/.m2/repository/edu/ucar/grib/4.5.5/grib-4.5.5.jar!/resources' does not exist
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.checkFileRequirements(FileUtils.java:1405) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1368) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1268) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1237) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.daemon.supervisor$fn__9436.invoke(supervisor.clj:1178) ~[storm-core-1.0.2.jar:1.0.2]
	at clojure.lang.MultiFn.invoke(MultiFn.java:243) ~[clojure-1.7.0.jar:?]
	at org.apache.storm.daemon.supervisor$mk_synchronize_supervisor$this__9163$fn__9181.invoke(supervisor.clj:579) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.daemon.supervisor$mk_synchronize_supervisor$this__9163.invoke(supervisor.clj:578) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.event$event_manager$fn__8735.invoke(event.clj:40) [storm-core-1.0.2.jar:1.0.2]
	at clojure.lang.AFn.run(AFn.java:22) [clojure-1.7.0.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
```

**Comment by ldkz2524** [comment]
same issue here. Can't seem to add tika and i previously removed flux due to the @sageeki 's  issue.

`java.io.FileNotFoundException: Source 'file:\C:\Users\Dongkeun\.m2\repository\edu\ucar\grib\4.5.5\grib-4.5.5.jar!\resources' does not exist
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.checkFileRequirements(FileUtils.java:1405) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1368) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1268) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.shade.org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1237) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.daemon.supervisor$fn__9436.invoke(supervisor.clj:1178) ~[storm-core-1.0.2.jar:1.0.2]
	at clojure.lang.MultiFn.invoke(MultiFn.java:243) ~[clojure-1.7.0.jar:?]
	at org.apache.storm.daemon.supervisor$mk_synchronize_supervisor$this__9163$fn__9181.invoke(supervisor.clj:579) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.daemon.supervisor$mk_synchronize_supervisor$this__9163.invoke(supervisor.clj:578) ~[storm-core-1.0.2.jar:1.0.2]
	at org.apache.storm.event$event_manager$fn__8735.invoke(event.clj:40) [storm-core-1.0.2.jar:1.0.2]
	at clojure.lang.AFn.run(AFn.java:22) [clojure-1.7.0.jar:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]`

**Comment by jnioche** [comment]
@ldkz2524 please use the storm command for now. I reported the issue in Storm but there hasn't been progress on that front. Thanks!

**Comment by normansuesstrunk** [comment]
i can run the Class locally in Eclipse with the following vm arguments: 
```
-conf crawler-conf.yaml -local
```
This seems to work fine so far.

**Comment by jnioche** [comment]
Fixed by upgrading to Storm 1.0.3 3bf149f
